src/webui/www/private/scripts/contextmenu.js

```diff
@@ -413,6 +438,42 @@ var TorrentsTableContextMenu = new Class({
             }
             categoryList.appendChild(el);
         });
+    },
+
+    updateTagsSubMenu: function(tag_list) {
+        const tagList = $('contextTagsList');
+        tagList.empty();
```

Comment:
where is `empty()` defined? and what does it do?

Reply:
Yep, it's mootools. Fixed.